cmd/cmd.go

```diff
@@ -66,8 +66,17 @@ func distroSupportsReExec() bool {
 	if !release.OnClassic {
 		return false
 	}
-	switch release.ReleaseInfo.ID {
-	case "fedora", "centos", "rhel", "opensuse", "suse", "poky", "arch", "manjaro", "lirios":
+	var contains = func(haystack []string, needle string) bool {
+		for _, s := range haystack {
+			if s == needle {
+				return true
+			}
+		}
+		return false
+
+	}
+	switch {
```

Comment:
The switch makes little sense here, can you make that into a normal if statement now please?

Reply:
This is still checking more than one conditional, so the switch makes sense still, I think.